File: src/main/java/com/nytdacm/oa/OaApplication.java
package com.nytdacm.oa;

import cn.dev33.satoken.jwt.SaJwtUtil;
import com.nytdacm.oa.config.CustomSaJwtTemplate;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class OaApplication {

    public static void main(String[] args) {
        SaJwtUtil.setSaJwtTemplate(new CustomSaJwtTemplate());
        SpringApplication.run(OaApplication.class, args);
    }

}

File: oa-app/src/main/kotlin/com/nytdacm/oa/OaApplication.kt
package com.nytdacm.oa

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableCaching
open class OaApplication

fun main(args: Array<String>) {
    runApplication<OaApplication>(*args)
}
